project/plugins.sbt

```diff
@@ -3,9 +3,8 @@ resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases
 resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
 resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.0.0")
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.1.0")
-addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.9.0")
```

Comment:
not in this file, but in `AppDependencies` if you stick `% "Test"` at the end of the test dependencies then that would get rid of some of the PR bot comments I think